spec/models/concerns/labware_creators/donor_pooling_calculator_spec.rb

```diff
@@ -0,0 +1,187 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+class TestPoolingClass
+  include LabwareCreators::DonorPoolingCalculator
+end
+
+RSpec.describe LabwareCreators::DonorPoolingCalculator do
+  let(:test_pooling_class) { TestPoolingClass.new }
```

Comment:
test_pooling_class returns an instance of TestPoolingClass but the name is misleading; test_pooling_instance ?

Reply:
changed